src/main/java/com/faforever/client/game/GameTileController.java

```diff
@@ -97,14 +105,29 @@ public void setGame(Game game) {
         game.mapFolderNameProperty());
 
     JavaFxUtil.bind(gameMapLabel.textProperty(), mapNameBinding);
+
     numberOfPlayersLabel.textProperty().bind(createStringBinding(
         () -> i18n.get("game.players.format", game.getNumPlayers(), game.getMaxPlayers()),
         game.numPlayersProperty(),
         game.maxPlayersProperty()
     ));
-    mapImageView.imageProperty().bind(createObjectBinding(
```

Comment:
Why was this dropped? I can not spot an equivalent replacement anywhere?

Reply:
Cause this code was present 2 times in exactly the same form.

Reply:
Show me the 2nd rime please can not find it

Reply:
found it, true this really is redudant

Reply:
gets overwritten by the second one